chart/tugger/values.yaml

```
@@ -93,7 +94,7 @@ livenessProbe:
   httpGet:
     scheme: HTTPS
     path: /ping
-    port: 443
+    port: *port # must match service.port
```

If you use the port name here you won't need to warning comment.
```suggestion
    port: https
```

Cool trick with the alias/reference though.